tests/testthat/error/test-type-data-frame.txt

```diff
@@ -33,3 +33,37 @@ i Falling back to <data.frame>.
 [1] mpg  cyl  disp qsec vs   am   hp   drat wt  
 <0 rows> (or 0-length row.names)
 
+> invisible(vec_rbind(foo, data.frame(), foo))
+Warning: Can't combine <foo> and <data.frame>.
+i Convert all inputs to the same class to avoid this warning.
+i See <https://vctrs.r-lib.org/reference/faq-warning-convert-inputs.html>.
+i Falling back to <data.frame>.
```

Comment:
I think this bullet should come first

Reply:
Or maybe the first sentence should be `Can't combine <foo> and <data.frame>; falling back to <data.frame>`.

Reply:
The fallback is now mentioned in the header.